test/library/etcd.go

```diff
@@ -0,0 +1,149 @@
+package library
```

Comment:
Why a new pkg? Could we move it to `test/e2e-encryption`?

Reply:
This is not a new package.  Existing test helpers are stored there.

Reply:
ok, sorry, didn't notice.